source/_layouts/layout.html

```diff
@@ -211,8 +221,16 @@
             new Main();
 
             new Search(projectSlug, versionSlug);
+
+            function googleTranslateElementInit() {
+                new google.translate.TranslateElement(
+                    {pageLanguage: 'en'}, 'google_translate_element'
+                );
+            }
         </script>
 
+        <script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
```

Comment:
I would always use HTTPS instead of making it protocol-relative. there is no reason to use HTTP when HTTPS is available.